### ISSUE TITLE
libusb: Pipe error

### ISSUE THREAD

**Issue by Quorsor** [Genymobile/scrcpy#4295]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 11
 - scrcpy version: 2.1
 - installation method: chocolatey
 - device model: Samsung A23 5G
 - Android version: 13

**Describe the bug**
The error occurs when I try to use OTG mode. Using the keyboard and mouse is fine when I use the ADB but trying to use 'scrcpy --otg` or any other option (referring to `--hid-mouse` and `--hid-keyboard`) ends up with a console output like this:

```
C:\Users\Admin>scrcpy --otg
scrcpy 2.1 <https://github.com/Genymobile/scrcpy>
INFO: Killing adb server (if any)...
INFO: USB device: 200901010001 (04f2:b72b) Chicony Electronics Co.,Ltd. HD User Facing
WARN: On this platform, libusb does not have hotplug capability; device disconnection will not be detected properly
ERROR: REGISTER_HID: libusb error: Pipe error
WARN: Register HID keyboard failed
```

**Comment by Quorsor** [comment]
It worked, thanks.

Didn't completely do the fix because somehow I can't install Google's WinUsb drivers on my phone but the default ADB device driver works. Is there a way to do this without having to use USB debugging on android? Some apps restrict themselves if  developer mode is turned on.

**Comment by rom1v** [comment]
Yes, `--otg` work with USB debugging turned off (except it's complicated on Windows because drivers may not detect the USB device if USB debugging is off, but on other platforms it works).
